src/ViewUser.js

```diff
@@ -262,6 +263,21 @@ class ViewUser extends React.Component {
     });
   }
 
+  buildRecords(recordsLoaded) {
```

Comment:
Please, avoid a mutation of the input props. This makes the code difficult to debug. Also, I'd like to propose a solution for your consideration:
```
recordsLoaded.map((record) => {
  const {
    item,
    item: {contributors},
  } = record;

  if (isArray(contributors)) {
    return {
      ...record,
      item: {
        ...item,
        contributors: contributors
          .map((currentContributor) => currentContributor.name)
          .join('; ')
      }
    };
  }
})
```

Reply:
@Yurii-Danylenko Thats a great solution. Added in the new logic